docs/Options.md

```diff
@@ -0,0 +1,501 @@
+  aws           # Amazon Web Services section
+  venv          # virtualenv section
+  conda         # conda virtualenv section
+  pyenv         # Pyenv section
+  dotnet        # .NET section
+  ember         # Ember.js section
+  kubecontext   # Kubectl context section
+  exec_time     # Execution time
+  line_sep      # Line break
+  battery       # Battery level and status
+  vi_mode       # Vi-mode indicator
+  jobs          # Backgound jobs indicator
+  exit_code     # Exit code section
+  char          # Prompt character
+)
+```
+
+You can also add items to the right side prompt by specifying them in the `SPACESHIP_RPROMPT_ORDER` option.
+
+### Prompt
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACESHIP_PROMPT_ADD_NEWLINE` | `true` | Adds a newline character before each prompt line |
+| `SPACESHIP_PROMPT_SEPARATE_LINE` | `true` | Make the prompt span across two lines |
+| `SPACESHIP_PROMPT_PREFIXES_SHOW` | `true` | Show prefixes before prompt sections or not |
+| `SPACESHIP_PROMPT_SUFFIXES_SHOW` | `true` | Show suffixes before prompt sections or not |
+| `SPACESHIP_PROMPT_DEFAULT_PREFIX` | `via ` | Default prefix for prompt sections |
+| `SPACESHIP_PROMPT_DEFAULT_SUFFIX` | ` ` | Default suffix for prompt section |
+
+### Char
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACESHIP_CHAR_SYMBOL` | `➜ ` | Prompt character to be shown before any command |
+| `SPACESHIP_CHAR_PREFIX` | ` ` | Prefix before prompt character |
+| `SPACESHIP_CHAR_SUFFIX` | ` ` | Suffix after prompt character |
+| `SPACESHIP_CHAR_COLOR_SUCCESS` | `green` | Color of prompt character if last command completes successfully |
+| `SPACESHIP_CHAR_COLOR_FAILURE` | `red` | Color of prompt character if last command returns non-zero exit-code |
+| `SPACESHIP_CHAR_COLOR_SECONDARY` | `yellow` | Color of secondary prompt character |
+
+### Time (`time`)
+
+Disabled as default. Set `SPACESHIP_TIME_SHOW` to `true` in your `.zshrc`, if you need to show time stamps.
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACESHIP_TIME_SHOW` | `false` | Show time (set to `true` for enabling) |
+| `SPACESHIP_TIME_PREFIX` | `at ` | Prefix before time section |
+| `SPACESHIP_TIME_SUFFIX` | `$SPACESHIP_PROMPT_DEFAULT_SUFFIX` | Suffix after time section |
+| `SPACESHIP_TIME_COLOR` | `yellow` | Color of time section |
+| `SPACESHIP_TIME_FORMAT` | `false` | Custom date formatting [ZSH date formats](http://zsh.sourceforge.net/Doc/Release/Prompt-Expansion.html#Date-and-time) |
+| `SPACESHIP_TIME_12HR` | `false` | Format time using 12-hour clock (am/pm) |
+
+### Username (`user`)
+
+Username is shown only when it's not the same as `$LOGNAME`, when you're connected via SSH or when you're root. Root user is highlighted in `SPACESHIP_USER_COLOR_ROOT` color (red as default).
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACESHIP_USER_SHOW` | `true` | Show user section |
```

Comment:
Are you planning to add details that there are 4 allowed values for this variable and describe their meaning, and if so, where? Currently I see `true`, I would guess that the value could also be `false`, but I wouldn't know about the existence of `always` or `needed`.

Reply:
Sure. WIP.

Reply:
Hm I didn't ask correctly, I was meaning to ask this: where are you planning to document all available options? Directly here, or somewhere in a different section of the documentation? A nested table of some kind?

Reply:
I think directly here. I'll figure out something.